src/views/htmlcontent/src/docs/index.html

```diff
@@ -3,7 +3,7 @@
 
 <h2 id="7">Version 0.3.0</h2>
 <ul>
-<li>Release date: February, 2016</li>
+<li>Release date: February 24, 2016</li>
```

Comment:
>li>Release date: February 24, 2016< [](start = 1, length = 35)

still haven't removed this file yet?  we should switch to popping up a web page for the next release.